packages/app/src/app/pages/Dashboard/Components/TemplatesRow/TemplatesRow.tsx

```diff
@@ -38,6 +38,10 @@ export const TemplatesRow: React.FC = () => {
     actions.modals.newSandboxModal.close();
   };
 
+  React.useEffect(() => {
+    actions.dashboard.getAllFolders();
+  }, [actions.dashboard, activeTeam]);
```

Comment:
The sidebar already makes this call, I'm not sure if we should have this or if the change in line 19 would be enough.

Reply:
Would this be needed here? I see it is already called twice, once from the sidebar and once from the page, will check if that's ok in a separate PR, but I wouldn't add this.

Reply:
I imagine we want to call this when the `TemplatesRow` is being used on its own - ~is the `SideBar` always rendered, e.g. on mobile?~ On mobile the `SideBar` is rendered too, so it's always being called.

The only reason we would want to call this _again_ is when we open a page with `TemplatesRow`, so potential newly created templates are fetched. It isn't the most glamorous approach though, and probably doesn't happen very often.